### ISSUE TITLE
Clarify Readme

### ISSUE THREAD

**Issue by Martyniuk** [uroslazarevic/repo-browser#1]
Hello Uros,

Please clarify with what values **.env** file you are expecting from user(developer) to populate.

**Comment by uroslazarevic** [comment]
Hello @Martyniuk ,

Thanks for openning the issue.
This [PR](https://github.com/uroslazarevic/repo-browser/pull/2) should solve the issue. Now `README.md` should be more clear and project setup should be more straightforward.

**Comment by Martyniuk** [comment]
The issue itself is not in having empty values in **.env** file but in providing descriptive meaning what and why smthing is used there.

Example:
- REACT_APP_USER_TOKEN - your [GitHub's PAT](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) for access to GitHub's public API.
- REACT_APP_CLIENT_ID - your GitHub used ID, find it here(link to where take it from)

In other words - there is a need for instructions on how to use your project & populating sensitive information is not a solution 😬

**Comment by uroslazarevic** [comment]
@Martyniuk ,

Gotcha, thanks for the clarification. I thought this info wasn't communicated to you in the first place.
I agree that the suggested change will be helpful in better understanding of used environment variables.

I will leave `.env` vars exposed, as they are already present in commit history and my OAuth app will be deprecated in the near future.

Thanks

**Comment by uroslazarevic** [comment]
Please feel free to close the issue if latest PR addresses it.

**Comment by Martyniuk** [comment]
@uroslazarevic,
The point is - your Readme should contain simple steps and descriptions that user (developer) may follow in order to run the project. 

Having this step is crucial еaking into consideration that you have BE proxy with Its own `.env` variables.

Exposed `.env` variables dont help in running your project. It is blocking it. Please see screenshot below

![Screenshot 2022-10-03 at 16 51 08](https://user-images.githubusercontent.com/12961943/193786745-e8f53d99-4cfb-4371-a68c-365bac3be1b3.png)

P.S. Actually, there is no need in BE proxy, please check [this link](https://docs.github.com/en/graphql/guides/forming-calls-with-graphql#authenticating-with-graphql)

**Comment by uroslazarevic** [comment]
Hello @Martyniuk ,

I've seen the docs for setting temporary access token. I've decided to implement single sign-on via GitHub login button, because it is more convenient than manually creating [PAT](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) and setting it in .env file. 

When using GitHub login API, there is need for backend proxy to avoid CORS issues. For example, this [thread](https://github.com/axios/axios/issues/2034) mentions it.
